### ISSUE TITLE
test demo

### ISSUE THREAD

**Issue by leowl0924** [Shyandram/CPGA-Net_Plus#1]
Hi, set params as follows and run demo.py. After running, the image is black, as if it has been clipped. Is it due to a post-processing error?
![image](https://github.com/user-attachments/assets/44815bd2-2af6-4afa-9cf3-7d38602c3ad7)
![image](https://github.com/user-attachments/assets/8d82c68e-458c-49a4-b516-9bbf26af82c7)

**Comment by Shyandram** [comment]
Turn "**--efficient**" to **False**, it is not for CPGA-Net+ but for CPGA-Net (original). 
BTW, we’ll release a more efficient version soon, so stay tuned!

**Comment by leowl0924** [comment]
Thanks for your reply. I'm looking forward to your new release.

**Comment by leowl0924** [comment]
BTW, the DGF is CPGA-Net+? 
I see that there is also a pytorch project on your homepage, what is the difference with this project?

**Comment by Shyandram** [comment]
[CPGA-Net+](https://arxiv.org/abs/2409.05274): **This repo,** Extended version of CPGA-Net
[CPGA-Net](https://arxiv.org/abs/2402.18147): [repo](https://github.com/Shyandram/CPGA-Net-Pytorch.git)
[CPGA-Net (DGF)](https://arxiv.org/abs/2402.18147): Efficient version of CPGA-Net via reducing resolution

This repo contains both CPGA-Net and CPGA-Net+, with a primary focus on CPGA-Net+ (the extended version).

**Comment by leowl0924** [comment]
> [CPGA-Net+](https://arxiv.org/abs/2409.05274): **This repo,** Extended version of CPGA-Net [CPGA-Net](https://arxiv.org/abs/2402.18147): [repo](https://github.com/Shyandram/CPGA-Net-Pytorch.git) [CPGA-Net (DGF)](https://arxiv.org/abs/2402.18147): Efficient version of CPGA-Net via reducing resolution
> 
> This repo contains both CPGA-Net and CPGA-Net+, with a primary focus on CPGA-Net+ (the extended version).

OK, thanks. I will depoly and test it.

**Comment by jacksongoode** [comment]
Is it possible to infer this model in real time on a video stream?

**Comment by Shyandram** [comment]
Yes, the speed is sufficient for real-time systems. Although it depends on your devices, most can achieve real-time or near-real-time inference. 
I will update the inference code & weights based on our [previous paper update](https://arxiv.org/abs/2409.05274) within a few days.

Sorry for the late reply.
